infrastructure/demo/bastion.tf

```
@@ -0,0 +1,16 @@
+module "bastion_host" {
+  source                          = "./bastion"
+  bastion_sg_name                 = "db-bastion-sg"
+  bastion_instance_profile_name   = "db-bastion-instance-profile"
+  bastion_instance_role_name      = "db-bastion-instance-role"
+  bastion_server_name             = "db-bastion-host"
```

Should these names also be dynamically defined like the following?
```suggestion
  bastion_sg_name               = "${var.project_name}-${var.environment}-db-bastion-sg"
  bastion_instance_profile_name = "${var.project_name}-${var.environment}-db-bastion-instance-profile"
  bastion_instance_role_name    = "${var.project_name}-${var.environment}-db-bastion-instance-role"
  bastion_server_name           = "${var.project_name}-${var.environment}-db-bastion-host"
```